tests/pingpong/host/host.cpp

```diff
@@ -24,11 +26,11 @@ void Pong(const char* in, char* out)
         {
             got_pong = true;
         }
-        strcpy(out, in);
+        strcpy_s(out, BUF_LENGTH, in);
```

Comment:
This looks dangerous since the API in line 19 does not guarantee that out is BUF_LENGTH.  Instead change the prototype so the length is a 3rd parameter in line 19.

Reply:
Thanks! I've changed the one in tests/pingpong-shared as well!